.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][<jira-issue-id>]: <short-description-of-the-pr> --->
```

Comment:
Re: `<jira-issue-id>`
I usually use the OSDOCS card, but idk if everyone does the same thing. Do we want to specify a preferred project format?

Reply:
Is card the right term? I was trying to specify the format in the docs manual, which specifies the _OSDOCS-<etc>_ format.

Reply:
Card is what I hear most often used as a generic (i.e., that would cover a feature, an RFE, or an issue). But I have never looked it up or asked a Jira expert :slightly_smiling_face: I forgot docs manual has `OSDOCS-####` - given that it does, maybe the template should suggest that specifically then. So:
```suggestion
<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->
```
maybe?